piker/brokers/binance.py

```diff
@@ -64,12 +63,14 @@
     ('low', float),
     ('close', float),
     ('volume', float),
-    ('close_time', int),
-    ('quote_vol', float),
-    ('num_trades', int),
-    ('buy_base_vol', float),
-    ('buy_quote_vol', float),
-    ('ignore', float)
+    # XXX: don't need these in shm history right?
```

Comment:
@guilledk I was gonna ask, I don't think we need this for now in the historical data right?

I don't really understand a couple of the fields actually.
Maybe we should make an issue to dig in later?

Reply:
Yeah I just put them for completeness but most are not necessary, I'll just leave the comment